src/Client.php

```diff
@@ -118,4 +122,17 @@ protected function getHttpClientBuilder()
     {
         return $this->httpClientBuilder;
     }
+
+    private function getApiClientVersion()
+    {
+        try {
+            if ($version = \Composer\InstalledVersions::getVersion('private-packagist/api-client')) {
```

Comment:
No need for the if

Reply:
I believe this still needs the if check because it might returns null

Reply:
If it is null then you return null anyway further below. Which will allow you to remove one of the return null